core/server/helpers/img_url.js

```diff
@@ -0,0 +1,14 @@
+// # Basically the same as {{content words="0"}} Content Helper
+// This is a temporary hack to enable lts themes to be forward compatible
+
+var hbs             = require('express-hbs'),
+    downzero        = require('../utils/downzero'),
+    content;
+
+content = function () {
+    return new hbs.handlebars.SafeString(
+        downzero(this.html)
```

Comment:
It is my misunderstanding of the `{{img_url}}` helper that is incorrect. I thought the idea was to mock the old hack of `{{content words="0"}}` but this is a completely different concept.

I will update the PR accordingly, maybe even get it to pass tests 😉